README.md

```diff
@@ -1,2 +1,343 @@
+<<<<<<< HEAD
```

Comment:
Please, remove the merge conflict tags, line 1, 4 and 343.

Reply:
Oh yeah, comes from an interactive rebasing... I didn't look into the file afterwards.